### PR TITLE
chore(deps): bump PF3 deps

### LINKED ISSUE #2889: Update dependencies
https://github.com/patternfly/patternfly-react/network/alerts

### LINKED ISSUE #2906: Security alerts showing up in product repos
These security alerts are showing up in Cost Management, partly due to PatternFly dependencies.

Screen Shot 2019-09-11 at 1 40 51 PM

See https://github.com/project-koku/koku-ui/issues/976

### PR DESCRIPTION
**Pull request by redallen**
**What**: Bump our version of PF3 core. Bump version of Storybook and fix its webpack config. Use a consistent version of lodash that's up to date. Closes #2889. Closes #2906.

**Additional issues**: The security tab of Github (and the workings of `npm audit`) is complicated. Outstanding vulnerabilities actually exported to our static sites should all be closed as a part of this PR.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -95,9 +93,8 @@
     "start": "yarn start:pf3",
     "start:cypress": "lerna run cypress:open",
     "start:demo-app": "lerna run start:demo-app --stream",
-    "start:pf3": "concurrently \"yarn storybook\" \"yarn storybook:openurl\"",
+    "start:pf3": "yarn storybook",
     "start:pf4": "yarn build:pf4 && lerna run develop --parallel --stream",
-    "storybook:openurl": "node ./storybook/openBrowser.js",
```
Comment (dlabaj) [inline package.json]: Is this not needed anymore?

File: storybook/webpack.config.js [hunk 2]
```diff
@@ -58,10 +48,11 @@ module.exports = (baseConfig, env, defaultConfig) => {
...
 
-  defaultConfig.resolve.alias = {
-    ...defaultConfig.resolve.alias,
+  config.resolve.alias = {
+    ...config.resolve.alias,
     'patternfly-react': path.resolve(__dirname, '../packages/patternfly-3/patternfly-react/src')
   };
 
-  return defaultConfig;
+  console.log('config', config.module.rules)
```
Comment (dlabaj) [inline storybook/webpack.config.js]: Do we need to log this ... can it be removed.

File: package.json (hunk 1 above)
Comment (redallen) [reply package.json]: Nope, the new storybook opens by itself and prints a message to the console with the link.

File: storybook/webpack.config.js (hunk 2 above)
Comment (redallen) [reply storybook/webpack.config.js]: Good catch, thanks.

**Comment by dlabaj** [comment]
@redallen Looks like the build is failing.

**Comment by redallen** [comment]
@dlabaj @tlabaj Can we get this in before we merge anything else? Any change will cause conflicts.

**Pull request merged**